### PR TITLE
Rework consumers

### PR DESCRIPTION
**Pull request by VLiamin**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/918

### CODE REVIEW

File: src/FileService.Business/Commands/File/EditFileCommand.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+using Microsoft.AspNetCore.JsonPatch;
+
+namespace LT.DigitalOffice.FileService.Business.Commands.File
+{
+  public class EditFileCommand : IEditFileCommand
+  {
+    private readonly IAccessValidator _accessValidator;
+    private readonly IFileRepository _fileRepository;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IResponseCreater _responseCreater;
```
Comment (MariyaSin) [inline src/FileService.Business/Commands/File/EditFileCommand.cs:23]: creatOr

File: src/FileService.Validation/EditFileRequestValidator.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          nameof(EditFileRequest.Name),
+          nameof(EditFileRequest.Content)
+        });
+
+      AddСorrectOperations(nameof(EditFileRequest.Name), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditFileRequest.Content), new List<OperationType> { OperationType.Replace });
+
+      #endregion
+
+      #region firstname
```
Comment (MariyaSin) [inline src/FileService.Validation/EditFileRequestValidator.cs:33]: ???

File: src/FileService.Validation/EditFileRequestValidator.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+      AddFailureForPropertyIf(
+        nameof(EditFileRequest.Name),
+        x => x == OperationType.Replace,
+        new Dictionary<Func<Operation<EditFileRequest>, bool>, string>
+        {
+        });
```
Comment (MariyaSin) [inline src/FileService.Validation/EditFileRequestValidator.cs:40]: could you remove it?

File: src/FileService/Startup.cs [hunk 4]
```diff
@@ -14,158 +17,168 @@
...
 
-            services.AddBusinessObjects();
+      services.Configure<TokenConfiguration>(Configuration.GetSection("CheckTokenMiddleware"));
+      services.Configure<BaseServiceInfoConfig>(Configuration.GetSection(BaseServiceInfoConfig.SectionName));
+      services.Configure<BaseRabbitMqConfig>(Configuration.GetSection(BaseRabbitMqConfig.SectionName));
 
-            services.AddControllers();
+      string connStr = Environment.GetEnvironmentVariable("ConnectionString");
+      if (string.IsNullOrEmpty(connStr))
+      {
```
[the remark this answers is not in the document]
Comment (VLiamin) [reply src/FileService/Startup.cs:77]: done

File: src/FileService.Business/Commands/File/EditFileCommand.cs (hunk 1 above)
Comment (VLiamin) [reply src/FileService.Business/Commands/File/EditFileCommand.cs]: done

File: src/FileService.Validation/EditFileRequestValidator.cs (hunk 2 above)
Comment (VLiamin) [reply src/FileService.Validation/EditFileRequestValidator.cs]: done

File: src/FileService.Business/Commands/File/EditFileCommand.cs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    private readonly IPatchDbFileMapper _mapper;
+
+    public EditFileCommand(
+      IAccessValidator accessValidator,
+      IHttpContextAccessor httpContextAccessor,
+      IResponseCreater responseCreater,
```
[the remark this answers is not in the document]
Comment (VLiamin) [reply src/FileService.Business/Commands/File/EditFileCommand.cs]: done

File: src/FileService.Validation/EditFileRequestValidator.cs (hunk 3 above)
Comment (VLiamin) [reply src/FileService.Validation/EditFileRequestValidator.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**